python/lsst/obs/decam/decamMapper.py

```diff
@@ -233,7 +241,11 @@ def std_raw(self, item, dataId):
 
     def std_dark(self, item, dataId):
         exp = afwImage.makeExposure(afwImage.makeMaskedImage(item))
-        exp.getCalib().setExptime(1.0)
+        rawPath = self.map_raw(dataId).getLocations()[0]
```

Comment:
Would there ever be possible that one tries to get a ``dark`` while ``raw`` of the same dataId is not available? (problems using ``map_raw`` in ``std_dark``? )